src/lib/components/workflow/workflows-summary-configurable-table.svelte

```diff
@@ -35,60 +27,47 @@
   };
 </script>
 
-<div class="flex flex-col">
-  <div
-    class="workflows-summary-configurable-table"
-    class:workflows-summary-configurable-table-empty={emptyAndNotLoading}
+<PaginatedTable
+  perPageLabel={translate('per-page')}
+  nextPageButtonLabel={translate('next-page')}
+  previousPageButtonLabel={translate('previous-page')}
+  pageButtonLabel={(page) => translate('go-to-page', { page })}
+  updating={$updating}
+  items={$workflows}
+  let:visibleItems
+>
+  <caption class="sr-only" slot="caption">
+    {translate('workflows')}
+  </caption>
+  <TableHeaderRow
+    onClickConfigure={openCustomizationDrawer}
+    workflows={visibleItems}
+    columnsCount={columns.length}
+    {empty}
+    slot="headers"
   >
-    <TableWrapper noPinnedColumns={pinnedColumns.length === 0}>
-      <Table pinned columns={pinnedColumns}>
-        <TableHeaderRow
-          pinned
-          {workflows}
-          {pinnedColumns}
-          {empty}
-          slot="headers"
-        >
-          {#each pinnedColumns as column}
-            <TableHeaderCell {column} />
-          {/each}
-        </TableHeaderRow>
-        {#each workflows as workflow}
-          <TableRow pinned {workflow}>
-            {#each pinnedColumns as column}
-              <TableBodyCell {column} {workflow} />
-            {/each}
-          </TableRow>
-        {/each}
-      </Table>
-      <Table columns={otherColumns} slot="unpinned-columns">
-        <TableHeaderRow
-          onClickConfigure={openCustomizationDrawer}
-          {workflows}
-          {pinnedColumns}
-          {empty}
-          slot="headers"
-        >
-          {#each otherColumns as column}
-            <TableHeaderCell {column} />
-          {/each}
-        </TableHeaderRow>
-        {#each workflows as workflow}
-          <TableRow {workflow}>
-            {#each otherColumns as column}
-              <TableBodyCell {column} {workflow} />
-            {/each}
-          </TableRow>
-        {/each}
-      </Table>
-    </TableWrapper>
-  </div>
-  {#if emptyAndNotLoading}
-    <TableEmptyState>
-      <slot slot="cloud" name="cloud" />
-    </TableEmptyState>
-  {/if}
-</div>
+    {#each columns as column}
+      <TableHeaderCell {column} />
+    {/each}
+  </TableHeaderRow>
+  {#each visibleItems as workflow}
+    <TableRow {workflow}>
```

Comment:
Looks like the multi-select isn't working as expected

![Screenshot 2023-08-14 at 10 13 09 AM](https://github.com/temporalio/ui/assets/15069288/1aa596d7-68d6-457e-a9e6-559649b518c6)

Reply:
**Update:** Unable to repro. this, but there is an existing bug with multi-select. Will address separately.